flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java

```diff
@@ -31,6 +33,8 @@ public class DorisOptions extends DorisConnectionOptions {
 
     private String tableIdentifier;
 
+    private String tableDdlCaseConversion;
```

Comment:
Is there no need to add this attribute here?

Reply:
> Is there no need to add this attribute here
![image](https://github.com/apache/doris-flink-connector/assets/46134044/f380f3b1-c10b-46f8-beb2-6b0ca66ddae9)
To use it here

Reply:
Just pass it in through the constructor method, similar to `userSchemaChange`?

Reply:
> userSchemaChange

![image](https://github.com/apache/doris-flink-connector/assets/46134044/4d4c27e9-ff28-463d-9fe3-861cc0f85463)
I borrowed tableIdentifier to write it

Reply:
Well, what I mean is that you can pass it directly to the JsonDebeziumSchemaSerializer method without modifying dorisoptions, because this is only used in cdc synchronization scenarios.